### PR TITLE
Update create account page

### PR DESCRIPTION
**Pull request by rosalieper**

### CODE REVIEW

File: src/components/Pages/CreateAccount.vue [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
   <v-container class="fill-height" fluid >
     <v-row align="center" justify="center">
-      <CreateAccountCard title="Lets get started" buttonText="Create Account"/>
+      <CreateAccountCard title="Sign up" buttonText="Create Account"/>
     </v-row>
       <v-row align="center" justify="center">
       <v-col class="needs-padding">
         <p>Already have an account?</p>
-        <router-link to="/login">Login now.</router-link>
+        <router-link to="/login">Log in</router-link>
```
Comment (tarrow) [inline src/components/Pages/CreateAccount.vue:9]: Not sure this should have lost the `.`

Comment (rosalieper) [reply src/components/Pages/CreateAccount.vue]: I changed it to a btn as per the AC.

File: src/components/Cards/CreateAccount.vue [hunk 2]
```diff
@@ -4,10 +4,16 @@
       <v-toolbar-title>{{title}}</v-toolbar-title>
     </v-toolbar>
     <v-card-text>
+      <p>
+        Wikibase.cloud is in a closed beta. If you'd like to receive an invite code,
+        <a href="https://lime.wikimedia.de/index.php/717538">
```
Comment (dati18) [inline src/components/Cards/CreateAccount.vue:9]: I think we should add `target="_blank"` here to open the link in a new tab. And also in other places with links (line 85 and 86 also)

File: src/components/Cards/CreateAccount.vue [hunk 3]
```diff
@@ -65,12 +71,11 @@
                     @click.stop
                     v-on="on"
                   >
-                    Terms of Use
+                    Terms of Use.
                   </a>
                 </template>
                 Opens in new window
```
Comment (dati18) [inline src/components/Cards/CreateAccount.vue:77]: `Open link in new tab` (like Chrome) sounds better technically

File: src/components/Cards/CreateAccount.vue (hunk 2 above)
Comment (rosalieper) [reply src/components/Cards/CreateAccount.vue:9]: the AC does not mention that. perhaps we should ask Charlie what she think about that.

File: src/components/Cards/CreateAccount.vue (hunk 3 above)
Comment (rosalieper) [reply src/components/Cards/CreateAccount.vue:77]: same here, we should ask UX

File: src/components/Cards/CreateAccount.vue (hunk 2 above)
Comment (rosalieper) [reply src/components/Cards/CreateAccount.vue:9]: As I understood Charlie, this is ok as is.

**Review by dati18 — APPROVED** [review]

**Comment by tarrow** [comment]
Is the fix to the login button visible on the preview? Right now I see the button below the text whereas in figma it is beside (with some space).

image

vs

image

**Comment by dati18** [comment]
Could you run it on you local env? Maybe the preview isn't updated

**Comment by tarrow** [comment]
Create Account page looks good! I didn't dig into it in great detail but I think a similar thing is needed on the log-in page?
image

**Comment by tarrow** [comment]
Login page button is now looking solid to me! Did you already checked the other design / layout features or does that still need to be done by someone?

**Pull request merged**